.circleci/config.yml

```diff
@@ -269,10 +270,16 @@ jobs:
       - run: *install_cached_packages
       - attach_workspace:
           at: .
-      - run: cover -write cover_db cover_db*
-      - run: cover -select_re '^(lib|script|t)/' -report html_minimal
+      - run:
+          name: Merge coverage databases
+          command: |
+            cover -write cover_db cover_db*
+            ls cover_db/*cov*
```

Comment:
Is this still necessary or debugging code that should be removed?

Reply:
I found it useful to find out if things work correctly.

Reply:
Of course. Question is if this should be kept in production code or remove it again as it was only useful for you at the time.